automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/SignInSpec.scala

```diff
@@ -13,14 +14,27 @@ class SignInSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Mat
 
   "A user with a registered account" - {
 
-      "should be able to log in and out" in {
-        // "harry": "harry.potter@quality.firecloud.org"
-        val student = Config.Users.testUser
-        withWebDriver { implicit driver =>
-          withSignInReal(student) { listPage =>
-            eventually { listPage.readUserEmail() shouldEqual student.email }
-          }
+    "should be able to log in and out as harry.potter" in {
+      // config is harded to get "harry": "harry.potter@quality.firecloud.org"
+      val harry = Config.Users.testUser
+      withWebDriver { implicit driver =>
+        withSignInReal(harry) { listPage =>
+          eventually { listPage.readUserEmail() shouldEqual harry.email }
         }
+        // making sure Sign Out worked
+        eventually { new SignInPage(Config.FireCloud.baseUrl).isOpen shouldEqual true }
       }
     }
+
+    "should be able to log in and out as ron.weasley" in {
+      // config is harded to get "harry": "harry.potter@quality.firecloud.org"
+      val ron = Config.Users.Students.getUserCredential("ron")
```

Comment:
same here-- let's add another testUser to the Config -- Config.Users.testUser2, and use that